ci/playbooks/content_provider/run.yml

```diff
@@ -31,3 +31,4 @@
           zuul:
             pause: true
           content_provider_registry_ip: "{{ content_provider_ip | default('nowhere') }}"
+          cifmw_operator_build_output: "{{ cifmw_operator_build_output }}"
```

Comment:
this appears to be undefied here

https://review.rdoproject.org/zuul/build/2836b71121c7445493eb4b7495ce95e0/log/job-output.txt#1319-1323

Reply:
as noted in slack there is a miss alignment on the key name

https://review.rdoproject.org/zuul/build/ad91003836bc477da6a65fad9886df45/log/zuul-info/inventory.yaml#33-39

the image key for the OpenStack-operator ahs the catalog URL and the base job is now expecting it to be called image_catalog https://review.rdoproject.org/r/c/config/+/48603/1/playbooks/crc/openstack.yaml#37

some eviter we need to revert https://review.rdoproject.org/r/c/config/+/48603/

or fix the cifmw_operator_build_output